test/source/mock/backend/backend-data.ts

```diff
@@ -38,35 +38,19 @@ export interface ReportedError {
 export class BackendData {
   public reportedErrors: ReportedError[] = [];
 
-  public clientConfigurationByAcctEmail: Dict<ClientConfiguration | HttpClientErr> = {};
+  public forceClientConfigurationForDomain: Dict<ClientConfiguration | HttpClientErr> = {};
```

Comment:
Not too sure about the `force` word, I don't see anything being forced

Reply:
let me change it to clientConfigurationForDomain